### PR TITLE
little bit of clean up

### PR DESCRIPTION
**Pull request by FloxD**
* removed log file
* autoformatting in intellij
* removed unused methods in HTTPSUtils
* set fields to final where intellij suggested it
* replaced string concat with stringbuilder where intellij suggested it
* replaced == with .equals for strings

### CODE REVIEW

File: Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java [hunk 1]
```diff
@@ -13,663 +28,641 @@
...
+                        if (isArchived) {
+                            if (member.getRoles().contains(prole)) {
+                                guild.removeRoleFromMember(userid, prole).queue(e -> {
+                                    jda.getTextChannelById(config.getString("log-channel")).sendMessage("<@&" + proleid + "> has been removed from <@" + userid + "> because the tourney is over!").queue();
+                                    logger.log("INFO", "Removed participant-role to " + userid);
+                                });
+                            }
+                            if (member.getRoles().contains(brole)) {
+                                guild.removeRoleFromMember(userid, brole).queue(e -> {
+                                    jda.getTextChannelById(config.getString("log-channel")).sendMessage("<@&" + broleid + "> has been removed from <@" + userid + "> because the tourney is over!").queue();
```
Comment (Floxinium) [inline Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java:158]: holy shit, the diff view of github makes this absolutely impossible to review, nice

File: Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java [hunk 2]
```diff
@@ -1,5 +1,20 @@
 package de.Ryeera.PaulBot;
 
+import de.Ryeera.libs.DragoLogger;
```
Comment (Floxinium) [inline Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java:3]: all the de.Ryeera dependencies here are not in any maven repository which makes it impossible to build this on someone else's machine

the logger at least could be replaced with log4j for example
with jsonutils and just utils idk

File: Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java [hunk 3]
```diff
@@ -13,663 +28,641 @@
...
+    private static JSONObject config = new JSONObject();
+    private static JSONObject matchrooms = new JSONObject();
+    private static JDA jda;
+
+    private static List<Role> adminRoles = new ArrayList<>();
+    private static List<Role> createRoles = new ArrayList<>();
+    private static List<Role> writeRoles = new ArrayList<>();
+    private static List<Role> readRoles = new ArrayList<>();
+
+    public static void main(String[] args) {
```
Comment (Floxinium) [inline Discord - Paul/src/main/java/de/Ryeera/PaulBot/Paul.java:49]: both the main method and the onMessageReceive method are huge
extracting both into separate classes are maybe even more classes probably makes sense in order to get some structure into this

**Comment by FloxD** [comment]
also i just was a bit bored, that's why i made these changes

**Pull request closed without merging**